docs/validate-stable-rc-builds.md

```diff
@@ -0,0 +1,178 @@
+
+# Stable rc review validation process
+Following steps to be performed when a stable rc review pushed on stable mailing list
+
+## Wait for stable rc review email
+Greg KH pushes on to stable rc trees and creates a tag
+then sends out email on stable ML with the list of patches included in that
+branch.
+
+You may check your inbox with give filters to identify the stable rc review
+for a given branch.
+
+## Build Test
+### Check for build is PASS
+open lkft.linaro.org website and check the build status
+From lkft.linaro.org you will re-directed to each branch builds for
+each device. If you see build status is in progress wait for the
+build to finish.
+
+https://ci.linaro.org/job/openembedded-lkft-linux-stable-rc-4.9/
```

Comment:
why is this here?

Reply:
updated documents and addressed your comments.